src/features/artReg/components/GeneralInfoStep.tsx

```diff
@@ -0,0 +1,470 @@
+import React, { useState } from 'react'
+import { Link } from 'react-router-dom'
+import { Formik, Form, Field } from 'formik'
+
+// import { useAppDispatch, useAppSelector } from '../../../redux/hooks'
+// import { FormControl } from '../../../common/components/FormControl/FormControl'
+import { Textarea, Button } from '../../common'
+import styles from './ArtRegForm.module.css'
+// import { IArtRegFormData, setFormData, setStep } from '../artRegSlice'
+import Select from 'react-select'
+
+// interface artRegFormStep1Values {
+//   title: string
+//   category: string
+//   copies: number
+// }
+
+export function GeneralInfoStep(): JSX.Element {
+  // const {
+  //   title,
+  //   // copies,
+  //   category,
+  //   externalProfile,
+  //   description,
+  //   // compensation,
+  // } = useAppSelector(state => state.artRegForm)
+
+  // const dispatch = useAppDispatch()
+  // const { register, handleSubmit } = useForm()
+
+  // !!!!
+  // function onSubmit(data: IArtRegFormData) {
+  //   dispatch(setFormData(data))
+  //   dispatch(setStep('ImageSelection'))
+  // }
+
+  const categoriesOptions = [
+    { value: 'option1', label: 'First' },
+    { value: 'option2', label: 'Second' },
+    { value: 'option3', label: 'Third' },
+    { value: 'option4', label: 'Fourth' },
+    { value: 'option5', label: 'Fifth' },
+    { value: 'option6', label: 'Sixth' },
+  ]
+
+  const compensationOptions = [
+    { value: 'royalty', label: 'Royalty' },
+    { value: 'option2', label: 'Percent' },
+    { value: 'option3', label: 'Big thanks' },
+  ]
+
+  const [copies, setCountOne] = useState(100)
+  const [royaltyPercent, setCountTwo] = useState(10)
+  const royaltyPercentString = `${royaltyPercent}%`
+
+  function showHideHeader() {
+    const headerElement = document.getElementById('headerPanel')
+    headerElement?.classList.toggle('hideHeaderPanel')
+  }
+
+  return (
+    <Formik
+      initialValues={{ title: '', category: '', copies: copies }}
+      onSubmit={(values, actions) => {
+        console.log({ values, actions })
+        alert(JSON.stringify(values, null, 2))
+        actions.setSubmitting(false)
+      }}
+    >
+      <Form>
+        <h1 className={styles.artRegFormTitle}>Input NFT data</h1>
+        <h2 className={styles.artRegFormSubTitle}>Description</h2>
+
+        <div className={styles.artRegFormProgressCircle}>
+          <svg
+            width='64'
+            height='64'
+            viewBox='0 0 64 64'
```

Comment:
Please use Tailwind classes.
I think it is better to keep svg files as assets instead of raw code.

Reply:
fixed

Reply:
Please resolve this conversation if it was fixed.